mmdeploy/__init__.py

```diff
@@ -2,7 +2,7 @@
 import importlib
 
 from mmdeploy.utils import get_root_logger
-from .version import __version__  # noqa F401
+from .version import *  # noqa F401
```

Comment:
hi, could you elaborate more on the motivation you use `version_info` instead of `mmdeploy.__version__`.

Reply:
Previously, the training referred to the "mmcv" library and compared versions using this. I encountered this problem when using 'mmdeploy' for reasoning.